solr/core/src/test/org/apache/solr/handler/export/TestExportWriter.java

```diff
@@ -684,13 +684,13 @@ public void testMultipleSorts() throws Exception {
     float_vals[5] = Float.MAX_VALUE;
     float_vals[6] = Float.MIN_VALUE;
     float_vals[7] = 1/3f; //0.33333334
-    float_vals[8] = 0.33333333f;
+    float_vals[8] = 0.33333333f; //0.33333334, intentional?
```

Comment:
what's this comment for?

Reply:
Floats aren't very precise. Sometimes you ask for one thing and get another!
I'm not sure if the value was intentionally equal to the previous value, or if this was an oversight.
```
jshell> 0.33333333f
$1 ==> 0.33333334
```

Reply:
interesting, if we remove one digit, it stays as `0.3333333f`. That would probably be more useful in the sorting test?